tests/ArabicdatetimeTest.php

```diff
@@ -42,6 +42,10 @@ public function testGetDays()
 
     public function testGetArabicDays()
     {
+        if (version_compare(PHP_VERSION, '5.6', '<')) {
```

Comment:
This assertion will not be worked on `php-5.4` and `php-5.5` versions because it seems that the `trans` is not worked.